src/Mod/Arch/ArchRoof.py

```diff
@@ -861,14 +1010,15 @@ def reject(self):
 
     def retranslateUi(self, TaskPanel):
         TaskPanel.setWindowTitle(QtGui.QApplication.translate("Arch", "Roof", None))
-        self.title.setText(QtGui.QApplication.translate("Arch", "Parameters of the profiles of the roof:\n* Angle : slope in degrees compared to the horizontal one.\n* Run : outdistance between the wall and the ridge sheathing.\n* Thickness : thickness of the side of roof.\n* Overhang : outdistance between the sewer and the wall.\n* Height : height of the ridge sheathing (calculated automatically)\n* IdRel : Relative Id for calculations automatic.\n---\nIf Angle = 0 and Run = 0 then profile is identical to the relative profile.\nIf Angle = 0 then angle is calculated so that the height is the same one as the relative profile.\nIf Run = 0 then Run is calculated so that the height is the same one as the relative profile.", None))
+        self.title.setText(QtGui.QApplication.translate("Arch", "Parameters of the roof profiles :\n* Angle : slope in degrees relative to the horizontal.\n* Run : horizontal distance between the wall and the ridge.\n* Thickness : thickness of the roof.\n* Overhang : horizontal distance between the eave and the wall.\n* Height : height of the ridge above the base (calculated automatically).\n* IdRel : Id of the relative profile used for automatic calculations.\n---\nIf Angle = 0 and Run = 0 then the profile is identical to the relative profile.\nIf Angle = 0 then the angle is calculated so that the height is the same as the relative profile.\nIf Run = 0 then the run is calculated so that the height is the same as the relative profile.", None))
```

Comment:
This entire text can be made readable inside the code if the lines are broken into multiple lines. In Python, text strings that are inside parentheses can be broken at any point, and they are concatenated automatically into a single string. A natural place to break the string is next to a newline character `"\n"`.

```py
_text = QtGui.QApplication.translate("Arch",
    "Parameters of the roof profiles :\n"
    "* Angle: slope in degrees relative to the horizontal.\n"
    "* Run: horizontal distance between the wall and the ridge.\n"
    "* Thickness: thickness of the roof.\n"
    "* Overhang: horizontal distance between the eave and the wall.\n"
    "* Height: height of the ridge above the base (calculated automatically).\n"
    "* IdRel: Id of the relative profile used for automatic calculations.\n"
    "---\n"
    "If Angle = 0 and Run = 0 then the profile is identical to the relative profile.\n"
    "If Angle = 0 then the angle is calculated so that the height is the same as "
    "the relative profile.\n"
    "If Run = 0 then the run is calculated so that the height is the same as "
    "the relative profile.",
    None)

self.title.setText(_text)
```